templates/stamp/clone-github-repository.sh

```diff
@@ -1,128 +0,0 @@
-#!/bin/bash
-#
-# Copyright (c) Microsoft Corporation. All Rights Reserved.
-# Licensed under the MIT license. See LICENSE file on the project webpage for details.
-#
-# todo: is this dead code or is it invoked through arm template's extension scripts? (aka oxa-tools/templates/stamp/stamp*json files)
```

Comment:
fyi, this is dead code. finally removing it because it reference github access tokens as well